### PR TITLE
Move IRC channel to Libera.Chat

### PR DESCRIPTION
**Pull request by slav0nic**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -451,5 +451,5 @@ def resig(app, what, name, obj, options, signature, return_annotation):
 linkcheck_ignore = [
     r'http://localhost:\d+',
     r'http://localhost',
-    r'https://webchat.freenode.net/#pyramid',  # JavaScript "anchor"
+    r'https://web.libera.chat/#pyramid',  # JavaScript "anchor"
```
Comment (slav0nic) [inline docs/conf.py:454]: check this pls

Comment (stevepiercy) [reply docs/conf.py:454]: Works for me!

**Comment by stevepiercy** [comment]
I'm going to merge this request. The build failures can be resolved in https://github.com/Pylons/pyramid/issues/3684

**Comment by stevepiercy** [comment]
Thank you!

Would you also please backport to the two branches `2.0-branch` and `1.10-branch`?

**Pull request merged**